src/iam/certmodules/certmodule.cpp

```diff
@@ -198,12 +201,17 @@ Error CertModule::ApplyCert(const String& pemCert, CertInfo& info)
         return AOS_ERROR_WRAP(err);
     }
 
+    err = TrimCerts(password);
```

Comment:
trimCert should be called before mHSM->ApplyCert. 
otherwise if it fails mStorage & mHSM will be inconsistent